uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/leave-request-ctrl_test.js

```diff
@@ -17,15 +16,15 @@
     'mocks/apis/option-group-api-mock',
     'mocks/apis/public-holiday-api-mock',
     'common/mocks/services/api/contact-mock',
-    'leave-absences/shared/controllers/leave-request-popup-controller',
+    'leave-absences/shared/controllers/sub-controllers/leave-request-ctrl',
```

Comment:
There should be a test file for `request-ctrl`and that should test `request-ctrl` alone. And `leave-request-ctrl_test` should test only the things which are unique to the `leave-request-ctrl`